test/RegisteredTest.t.sol

```diff
@@ -0,0 +1,29 @@
+pragma solidity ^0.8.13;
+
+import "./RegisteredAbstract.t.sol";
+
+contract RegisteredTest is RegisteredAbstract {
+
+    function testBubsyRugged() public {
```

Comment:
Funny, but the test name should describe more precisely what is that the test does :)